osu.Game/Screens/OnlinePlay/OnlinePlayScreen.cs

```diff
@@ -32,8 +32,9 @@ public abstract partial class OnlinePlayScreen : OsuScreen, IHasSubScreenStack
         // while leases may be taken out by a subscreen.
         public override bool DisallowExternalBeatmapRulesetChanges => true;
 
+        protected LoungeSubScreen LoungeSubScreen;
```

Comment:
```suggestion
        protected LoungeSubScreen LoungeSubScreen { get; private set; }
```

Also maybe rename to `LoungeScreen`? Or `Lounge`?